src/gsi/read_prepbufr.f90

```diff
@@ -2556,6 +2557,7 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
 !                need to find out gustoe
 !                gustoe=1.8
                  gustoe=1.0
+                 if ( l_rtma3d .and. oerr_gust > 0.0_r_kind ) gustoe = oerr_gust
```

Comment:
the same concern,  if i_gust_3dda=1 but oerr_gust is not set in the namelist , this line would be not executed that the user might be unable to find .   And, here, maybe i_gust_3dda is a better parameter to be tested than l_rtma3d.

Reply:
@TingLei-NOAA 
Hi Ting,
Similar as the setup for parameters of background error (co)variances for wind gust, in most cases, esp. in 2DRTMA run, the observation error of wind gust (gustoe) is the default value which is hardwired in GSI code (1.0 m/s). Usually user does not need to set this parameter. Even if user would like to tune the obs error of wind gust, it is only for 3drtma run (definitely not in any 2DRTMA run), that's why l_rtma3d is used here to make sure obs error of wind gust is changed only if it is 3DRTMA run.

I will following your suggestion to add a print-out of the observation error of wind gust to remind the user. 

Thank you!

-Gang

Reply:
@GangZhao-NOAA  Thanks for the explanation.  That is ok if it is restricted 3drtma ( in the future, if, say, rrfs need this, it could be changed ).   And no need to add a warning message, since it is in a loop and would produce maybe too many duplicated messages.

Reply:
@TingLei-NOAA Yes, definitely we can set up the observation of error of wind gust also for other DA systems (such as rrfsf-da) if gust is analyzed in that DA system. We can set different values of obs error of gust with respect to different DA systems.
Thank you!